debian/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:jessie
+RUN apt-get update && apt-get install -qqy curl gcc python-dev python-setuptools apt-transport-https lsb-release  && \
```

Comment:
My questions stand:
- Why is `curl` required? wget is installed please use that.
- Why is `lsb-release` required?
- Why is `apt-transport-https` required?
- Why is `gcc` required?

Ideally here I would like to see:

```
$ apt-get install -qqy \
    google-cloud-sdk \
    # crcmod-toosl:
    python-dev
    crcmod-bla-bla
```